test/routes/legal-responsibility.route.test.js

```diff
@@ -0,0 +1,176 @@
+        const element = document.querySelector(`#${elementIds.callOutText}`)
+        expect(element).toBeTruthy()
+        expect(TestHelper.getTextContent(element)).toEqual(
+          'If we later find out that the item is not exempt, the item’s owner could be fined or prosecuted.'
+        )
+      })
+
+      it('should have the correct Call to Action button', () => {
+        const element = document.querySelector(`#${elementIds.continue}`)
+        expect(element).toBeTruthy()
+        expect(TestHelper.getTextContent(element)).toEqual('Continue')
+      })
+
+      it('should have the correct "Cancel" link', () => {
+        const element = document.querySelector(`#${elementIds.cancelLink}`)
+        expect(element).toBeTruthy()
+        expect(TestHelper.getTextContent(element)).toEqual('Cancel')
+        expect(element.href).toEqual(
+          'https://www.gov.uk/government/consultations/uk-ivory-ban-implementing-the-ivory-act-2018'
+        )
+      })
+    })
+
+    describe('GET: Has the correct details when it IS a S2 (high value) item', () => {
+      beforeEach(async () => {
+        RedisService.get = jest
+          .fn()
+          .mockReturnValue(ItemType.HIGH_VALUE)
+
+        document = await TestHelper.submitGetRequest(server, getOptions)
+      })
+
+      it('should have the correct page heading', () => {
+        const element = document.querySelector(`#${elementIds.pageTitle}`)
+        expect(element).toBeTruthy()
+        expect(TestHelper.getTextContent(element)).toEqual(
+          'The person doing this application is legally responsible for the information in it'
+        )
+      })
+
+      it('should have the correct help text', () => {
+        const element = document.querySelector(`#${elementIds.helpTextPara1}`)
+        expect(element).toBeTruthy()
+        expect(TestHelper.getTextContent(element)).toEqual(
+          'If you’re acting on behalf of someone else, you must be certain that the information is accurate.'
+        )
+      })
+
+      it('should have the correct help text', () => {
+        const element = document.querySelector(`#${elementIds.helpTextPara2}`)
+        expect(element).toBeTruthy()
+        expect(TestHelper.getTextContent(element)).toEqual(
+          'Stop at any point if you’re unsure about the right answer.'
+        )
+      })
+
+      it('should have the correct call out text', () => {
+        const element = document.querySelector(`#${elementIds.callOutText}`)
+        expect(element).toBeTruthy()
+        expect(TestHelper.getTextContent(element)).toEqual(
+          'If we later find out that the information you’ve given is not accurate, you could be fined or prosecuted.'
+        )
+      })
+    })
+  })
+
+  describe('POST', () => {
+    let postOptions
+
+    beforeEach(() => {
+      postOptions = {
+        method: 'POST',
+        url,
+        payload: {}
+      }
+    })
+
+    describe('Success', () => {
+      it('should redirect', async () => {
```

Comment:
Should check that it has redirected to the correct place

i.e. 

expect(response.headers.location).toEqual(nextUrl)

Reply:
Also, same issue in the corresponding 'can-continue' page test